.devcontainer/devcontainer.json

```diff
@@ -1,17 +1,12 @@
 {
-  "name": "ComposerBuilder",
+  "name": "composer",
   "build": {
     "dockerfile": "Dockerfile",
     "context": ".."
   },
-  "mounts": [
-    "source=osbuild-share,target=/root/.local/share,type=volume"
-  ],
-  "runArgs": [
-    "--privileged"
```

Comment:
I'm afraid this leads to the workspace not being readable in the container (SE Linux and stuff, see also https://github.com/microsoft/vscode-remote-release/issues/1333).

Maybe someone else can try this out on native Linux as I presume you cannot reproduce on MacOS.

Reply:
I also use `osbuild-pipeline` and `osbuild` directly in the container and the latter needs `--privileged`.

Reply:
Oops! Sorry for missing both these use cases (especially the non-macos one ;) ). Fixed now.